### DISCUSSION TITLE
I have now tried to build a built-in display successfully.

### DISCUSSION THREAD

**Discussion by bfykk** [MonitorControl/MonitorControl#1223] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

I have now tried to build a built-in display successfully. 
At the same time, the brightness slider of the display is called out. 
https://bbs.pcbeta.com/viewthread-1947168-1-1.html
After using the software, you can control the brightness slider of the built-in display, but the backlight will not change. Can you improve the software to support the DDC dimming of the built-in display.
You can try the linked scheme to make improvements. If it can be realized, ordinary displays will be perfect.

**Comment by waydabber** [comment]
Hi there, sorry for responding so late. It depends on how did you build the built-in display and what control board it uses and whether the board supports DDC or not (sometimes it's just a matter of a firmware setting). Without the details it's a bit difficult to answer to this.
